pkg/scheduler/algorithm/priorities/image_locality.go

```diff
@@ -74,10 +76,23 @@ func totalImageSize(nodeInfo *schedulercache.NodeInfo, containers []v1.Container
 
 	imageSizes := nodeInfo.ImageSizes()
 	for _, container := range containers {
-		if size, ok := imageSizes[container.Image]; ok {
+		if size, ok := imageSizes[normalizedImage(container.Image)]; ok {
```

Comment:
normalizedImage is not enough :-)

Please make sure this check work for both `docker.io/library/nginx:1.7.9` or `nginx:1.7.9` in `nodeStatus`, and add test case for it.

Reply:
And after #64413, we don't need this workaround.

Reply:
Sounds good and applied the workaround as per:

>Please make sure this check work for both docker.io/library/nginx:1.7.9 or nginx:1.7.9 in nodeStatus, and add test case for it.